modules/aws/ecs_test.go

```diff
@@ -21,3 +23,33 @@ func TestEcsCluster(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, "terratest", *c2.ClusterName)
 }
+
+func TestEcsClusterWithInclude(t *testing.T) {
+	t.Parallel()
+
+	region := GetRandomStableRegion(t, nil, nil)
+
+	client := NewEcsClient(t, region)
+	c1, err := client.CreateCluster(&ecs.CreateClusterInput{
+		ClusterName: aws.String("terratest-extended"),
```

Comment:
You need to use `random.UniqueId` in the name. See [namespacing](https://terratest.gruntwork.io/docs/testing-best-practices/namespacing/) for more info.

Reply:
Ha, I was wondering about that, but as the existing test didn't use it, I opted to not do it as well.